### PR TITLE
Start project

### PR DESCRIPTION
**Pull request by lucasrcezimbra**
Generated using https://github.com/lucasrcezimbra/cookiecutter-pypackage

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+bump2version==0.5.11
+
```
Comment (lucasrcezimbra) [inline requirements_dev.txt:3]:
```suggestion
bump2version==0.5.11
```

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements-dev.txt
```
Comment (daniellbastos) [inline .github/workflows/python-app.yml:21]: Fix the filename to `requirements_dev.txt`

File: README.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+Contributions are welcome, feel free to open an Issue or Pull Request.
+
+Pull requests must be for the `develop` branch.
+
+```bash
+git clone https://github.com/HBN3tw0rk/dcache
+cd dcache
+git checkout develop
+python -m venv .venv
+pip install -r requirements-dev.txt
```
Comment (daniellbastos) [inline README.md:39]: Fix the filename to `requirements_dev.txt`

**Review by daniellbastos — CHANGES_REQUESTED** [review]
I left a comment with a little fix

File: .github/workflows/python-app.yml (hunk 2 above)
Comment (lucasrcezimbra) [reply .github/workflows/python-app.yml:21]: Good catch. Thanks.

```suggestion
        pip install -r requirements_dev.txt
```

File: README.md (hunk 3 above)
Comment (lucasrcezimbra) [reply README.md:39]:
```suggestion
pip install -r requirements_dev.txt
```

**Review by daniellbastos — APPROVED** [review]
LGTM

File: docs/readme.rst [hunk 4]
```diff
@@ -0,0 +1 @@
+.. include:: ../README.rst
```
Comment (lucasrcezimbra) [inline docs/readme.rst:1]: Change README to rst

File: docs/history.rst [hunk 5]
```diff
@@ -0,0 +1 @@
+.. include:: ../HISTORY.rst
```
Comment (lucasrcezimbra) [inline docs/history.rst:1]: Change CHANGELOG to rst

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+with open('CHANGELOG.md') as history_file:
+    history = history_file.read()
+
+requirements = []
+
+test_requirements = ['pytest']
+
+setup(
+    author="Lucas Rangel Cezimbra",
+    author_email='lucas@cezimbra.tec.br',
```
Comment (lucasrcezimbra) [inline setup.py:19]:
```suggestion
    author="HBN3tw0rk",
```

**Pull request merged**